aggregate/quality_of_life/access_to_jobs.py

```diff
@@ -0,0 +1,38 @@
+from curses import raw
+from turtle import pu
+from matplotlib.pyplot import axis
+import pandas as pd
+from internal_review.set_internal_review_file import set_internal_review_files
+from utils.assign_PUMA import clean_PUMAs, puma_to_borough
+
+
+def access_to_jobs(geography, write_to_internal_review=False):
+    indicator_col_name = "access_employment"
+    clean_df = load_clean_source_data(indicator_col_name)
+
+    final = clean_df.groupby(geography).sum()[indicator_col_name]
+    if write_to_internal_review:
+        set_internal_review_files(
+            [(final, f"access_employment.csv", geography)],
```

Comment:
not sure the format string is necessary here but this is really minor.

Reply:
oh yea lol you're correct I can fix that. pure habit I guess?